src/embind/embind.js

```diff
@@ -35,6 +35,9 @@ var LibraryEmbind = {
   $PureVirtualError__deps: ['$extendError'],
   $PureVirtualError: undefined,
   $GenericWireTypeSize: {{{ 2 * POINTER_SIZE }}},
+#if EMBIND_AOT
+  $InvokerFunctions: '<<< EMBIND_AOT_OUTPUT >>>',
```

Comment:
Does this work if EMBIND_AOT_OUTPUT is mulit-line?  Maybe backticks?

Reply:
The quotes get removed, so this becomes `var InvokerFunctions = <<< EMBIND_AOT_OUTPUT >>>;` and 
EMBIND_AOT_OUTPUT will be an object literal e.g. `{'3434': function(...}` so it should be fine